actions/opslevel/main.go

```diff
@@ -0,0 +1,164 @@
+// Copyright 2022 Outreach Corporation. All Rights Reserved.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/getoutreach/actions/internal/gh"
+	"github.com/getoutreach/actions/pkg/opslevel"
+	"github.com/getoutreach/actions/internal/slack"
+	"github.com/google/go-github/v43/github"
+	opslevelGo "github.com/opslevel/opslevel-go/v2022"
+	"github.com/pkg/errors"
+	actions "github.com/sethvargo/go-githubactions"
+	slackGo "github.com/slack-go/slack"
+)
+
+// Constant block for slack message formatted strings.
+const (
+	// slackMessageFmt is a formatted string that is meant to have the following data
+	// passed to it to build it:
+	//	- Project repository hyperlink
+	//      - Expected OpsLevel level
+	//      - Actual OpsLevel level
+	//	- Hyperlinked OpsLevel Maturity Report
+	//	- Appended message content (anything, or empty)
+	//
+	// All of this information can be found in the status event payload sent to the action.
+	slackMessageFmt = `Looks like` + "`%s`" + ` does not meet the specied level in OpsLevel.
```

Comment:
I'm not sure what "specied" means here; I'm guessing typo? You're also missing a space:
```suggestion
	slackMessageFmt = `Looks like ` + "`%s`" + ` does not meet the required level in OpsLevel.
```
I'd also go further and make this a little less casual:
```suggestion
	slackMessageFmt = `Your service ` + "`%s`" + ` does not meet the required level in OpsLevel.
```